tests/antares/services/local_services/test_study.py

```diff
@@ -1438,7 +1440,10 @@ def mock_error_in_sets_ini():
             raise CustomError("An error occurred while processing area can not be created")
 
         monkeypatch.setattr("antares.service.local_services.area_local._sets_ini_content", mock_error_in_sets_ini)
-        with pytest.raises(CustomError, match="An error occurred while processing area can not be created"):
+        with pytest.raises(
+            AreaCreationError,
+            match="Could not create the area test: Error during area creation: An error occurred while processing area can not be created",
```

Comment:
This is a really long msg, it should be shortened

Reply:
This is a combination of two levels of error messages raised in this particular test, as you see the two sets of colons
1) 'Could not create the area test:' this is built into AreaCreationError
2) 'Error during area create:' this is raised in create_area_local
3) The last bit is specific to this test

I have proceeded to delete 2 as I think it's superfluous.